tests/Doctrine/Tests/ORM/Functional/Ticket/GH9109Test.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\ManyToMany;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+/**
+ * @group GH-9109
+ */
+class GH9109Test extends OrmFunctionalTestCase
+{
+    public function testIssue(): void
+    {
+        $userFirstName = 'GH9109Test';
+        $userLastName  = 'UserGH9109';
+        $productTitle  = 'Test product';
+
+        $this->_schemaTool->createSchema(
+            [
+                $this->_em->getClassMetadata(GH9109User::class),
+                $this->_em->getClassMetadata(GH9109Product::class),
+            ]
+        );
```

Comment:
Please call `$this->_schemaTool->dropSchema(…)` in `tearDown()`

Reply:
done